ui/app/src/components/Host/Host.tsx

```diff
@@ -62,7 +62,7 @@ export function Host() {
 
     const h2g = getHostToGuestBus()
       .asObservable()
-      .pipe(map((action) => ({ ...action, craftercms: true, source: 'host' })))
+      .pipe(map((action) => ({ ...action, meta: { craftercms: true, source: 'host' } })))
```

Comment:
The cause: 
- https://github.com/craftercms/studio-ui/blob/33545768b3e4189b0f27c94aea72baa60e80ec15/static-assets/scripts/communicator.js#L252
- During recent changes, the meta object got lost and its props were added to the root payload object